src/main/java/seedu/realodex/model/person/HousingType.java

```diff
@@ -0,0 +1,84 @@
+package seedu.realodex.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.realodex.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Housing Type in the realodex.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidHousingType(String)}
+ */
+public class HousingType {
+
+    public static final String MESSAGE_CONSTRAINTS = "Housing type should be either 'HDB', 'CONDOMINIUM', "
+            + "'LANDED PROPERTY' or 'GOOD CLASS BUNGALOW'";
```

Comment:
Perhaps we can abbreviate good class bungalow to "GCB", Landed Property to "landed"? I am worried that this may be considered a feature flaw as the user has to type out so long

Reply:
If you want, this can be done in a new PR